src/main/java/thePackmaster/powers/rimworldpack/DupePower.java

```diff
@@ -0,0 +1,59 @@
+package thePackmaster.powers.rimworldpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DrawCardAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardQueueItem;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.adp;
+
+public class DupePower extends AbstractPackmasterPower {
```

Comment:
Looks to me like the strings for this power are missing -- I don't see anything in Powerstrings.json for it.

I do understand why this exists and why you didn't use the base game power that Duplication Pot uses -- I'm guessing you don't want the part where it goes away at the end of the turn, since that's missing but the rest of the code is basically the same.

Reply:
Yup it was the end of turn effect that I didn't want. And also whoops, I have no idea how I missed the power strings for this